apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,25 +154,14 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
```

Comment:
I've made some additional investigation regarding this removed functionality. And here some notes:

1. This is pretty flexible way of selection proper type of progress indicator to be shown to user, it depends on exact 'strategy' used for export (currently I've found 3 implementations: ``bt.js, sd.js & sim.js``). As soon as I wasn't able to quickly verify with UI team if there expected more than one type of progress indicator, I think that complete removal of this feature isn't a proper solution at least for this stage of project.
2. Simpliest and quickest way to use only ``'progressBar'`` type is to always assign ``true`` to ``determinativeProgress`` in this function ``_configureProgress()`` **and** placing appropriate comment here, something like: "usage of progress indicator type defined by strategy is disabled for RedSquare project according to current UX requirements".
3. My main concern is that UI team can relatively simply decide to use another type of progress indicator and in this case we will need to recover this functionality, so taking this into account I think that Mozilla won't allow such change without direct approve from MozUX team.
4. Anyway, even if (unlikely) such removal shall be done, it shall be performed for all 'strategies' including ``sms.js`` which was left 'as is' in current version of patch.